atuin-client/src/settings.rs

```diff
@@ -43,13 +43,32 @@ pub enum FilterMode {
     Directory = 3,
 }
 
+impl SearchMode {
+    pub fn as_str(&self) -> &'static str {
+        match self {
+            SearchMode::Prefix => "PREF",
+            SearchMode::FullText => "FULL",
+            SearchMode::Fuzzy => "FUZZ",
+        }
+    }
+
+    // Get next mode in order
+    pub fn next(&self) -> Self {
+        match self {
+            SearchMode::Prefix => SearchMode::FullText,
+            SearchMode::FullText => SearchMode::Fuzzy,
+            SearchMode::Fuzzy => SearchMode::Prefix,
+        }
+    }
+}
+
 impl FilterMode {
     pub fn as_str(&self) -> &'static str {
         match self {
-            FilterMode::Global => "GLOBAL",
+            FilterMode::Global => "GLOB",
             FilterMode::Host => "HOST",
-            FilterMode::Session => "SESSION",
-            FilterMode::Directory => "DIRECTORY",
+            FilterMode::Session => "SESS",
+            FilterMode::Directory => "DIR",
         }
```

Comment:
Why change this?

Reply:
Needed to fit `filtermode | searchmode` in the 10-ish chars width available in the left panel. Happy to revert once we design a better UI.

Reply:
Reverted.